typed-racket-test/succeed/flonum-typecheck.rkt

```diff
@@ -0,0 +1,123 @@
+#lang typed/racket
+(require racket/flonum
+         typed/rackunit)
+
+
+;;Flonum typechecking tests
+;; -FlZero -Fl
+(check-eq?
```

Comment:
I think the issue is that you were looking in the wrong file. This file is not a unit test (as we call it) but an integration test, as it checks the entire TR system e2e. The unittests for typechecking are located here: https://github.com/racket/typed-racket/blob/master/typed-racket-test/unit-tests/typecheck-tests.rkt. And those much more easily support the kind of test you are trying to write, and support negative tests.